bread/layout/components/forms/fields.py

```diff
@@ -34,8 +34,8 @@ def generate_widget_element(
     # parameters which are normally not required, when using a django form field
     # but can be filled in to create form fields independently from django form fields or
     # manually overriding values from the form field
-    widgetclass: Optional[
-        Type[BaseWidget]
+    widgetclass: Union[
+        Optional[Type[BaseWidget]], hg.Lazy
```

Comment:
I think this should be ``` Optional[ Union[ Type[ BaseWidget ], hg.Lazy ] ]```.

Reply:
The point being that the parameter is optional per se, not only a certain option of the parameter type.

Reply:
Yeah I think it's the same thing. `Optional[Type[BaseWidget]]` is the same as `Union[Type[BaseWidget], None]`.
So the typehint could also be `Union[ Type[ BaseWidget ], hg.Lazy, None]`. It's a matter of taste. But I will use your suggestion because it's not really worth a big discussion.

Reply:
Agreed, it is the same. But thanks for changing.